packages/patternfly-4/react-core/src/experimental/components/DataToolbar/DataToolbarItemWithChipGroup.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { DataToolbarItem, DataToolbarItemProps } from './DataToolbarItem';
+import { ChipGroup, Chip, ChipGroupToolbarItem } from '../../../components/ChipGroup';
+import { DataToolbarContext } from './DataToolbarUtils';
+import styles from '@patternfly/react-styles/css/components/DataToolbar/data-toolbar';
+import { getModifier } from '@patternfly/react-styles';
+
+export interface DataToolbarItemWithChipGroupProps extends DataToolbarItemProps {
+  /** An array of strings to be displayed as chips in the expandable content */
+  chips?: string[];
```

Comment:
This should be array of `React.ReactNode` in order to allow translations and custom components.